src/app/backend/auth/jwt/manager.go

```diff
@@ -0,0 +1,88 @@
+// Copyright 2017 The Kubernetes Dashboard Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package jwt
+
+import (
+	"crypto/rand"
+	"crypto/rsa"
+
+	authApi "github.com/kubernetes/dashboard/src/app/backend/auth/api"
+	"gopkg.in/square/go-jose.v2"
+	"k8s.io/apimachinery/pkg/util/json"
+	"k8s.io/client-go/tools/clientcmd/api"
+)
+
+// TODO(floreks): Should be retrieved from a secret
+var tokenSigningKey *rsa.PrivateKey
```

Comment:
it still should be in an instance, not a singleton to enable rotation, testing, etc